<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>-10</y>
      <width>371</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QPushButton" name="powerButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>510</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Power</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="powerOffScreen">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>231</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="displayScreen">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>231</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="frequencyLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>270</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="frequencyButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>460</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Frequency</string>
     </property>
    </widget>
    <widget class="QPushButton" name="waveformButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>460</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Waveform</string>
     </property>
    </widget>
    <widget class="QLabel" name="waveformLabel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>270</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel_20min">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel_40min">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel_60min">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="timeButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>370</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
    <widget class="QPushButton" name="increaseCurrentButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>420</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>increase</string>
     </property>
    </widget>
    <widget class="QPushButton" name="decreaseCurrentButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>420</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>decrease</string>
     </property>
    </widget>
    <widget class="QLabel" name="current_8">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>211</x>
       <y>231</y>
       <width>14</width>
       <height>19</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>19</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>19</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="current_5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>151</x>
       <y>231</y>
       <width>14</width>
       <height>19</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>19</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>19</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="current_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>111</x>
       <y>231</y>
       <width>14</width>
       <height>19</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>19</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>19</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="current_1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>231</y>
       <width>14</width>
       <height>19</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>19</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>19</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="current_10">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>251</x>
       <y>231</y>
       <width>14</width>
       <height>19</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>19</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>19</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="current_6">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>171</x>
       <y>231</y>
       <width>14</width>
       <height>19</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>19</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>19</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="current_9">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>231</x>
       <y>231</y>
       <width>14</width>
       <height>19</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>19</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>19</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="current_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>231</y>
       <width>14</width>
       <height>19</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>19</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>19</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="current_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>91</x>
       <y>231</y>
       <width>14</width>
       <height>19</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>19</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>19</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="current_7">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>191</x>
       <y>231</y>
       <width>14</width>
       <height>19</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>19</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>19</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QProgressBar" name="batteryBar">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>40</width>
       <height>12</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>12</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>12</height>
      </size>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="lowBatteryWarning_2P">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>201</width>
       <height>17</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>201</width>
       <height>17</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>201</width>
       <height>17</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(164, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lowBatteryWarning_5P">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>201</width>
       <height>17</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>201</width>
       <height>17</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>201</width>
       <height>17</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(128,128,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply To Skin</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>96</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply To Skin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chargeButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Charge</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="batteryLineEdit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16</width>
      <height>21</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QTableView" name="RecordView">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>351</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveRecordingButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>210</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Recording</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadRecordingButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>210</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Recordings</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
